src/test/resources/docker/yugabytedb-init.sql

```diff
@@ -0,0 +1,34 @@
+    CREATE DATABASE lbcat;
+    \c lbcat;
+    CREATE USER lbuser WITH PASSWORD 'LiquibasePass1';
+    GRANT ALL PRIVILEGES ON DATABASE lbcat TO lbuser;
+    GRANT ALL PRIVILEGES ON SCHEMA public TO lbuser;
+    DROP TABLE IF EXISTS authors;
+    CREATE TABLE authors (
+      id SERIAL,
+      first_name VARCHAR (50) NOT NULL,
+      last_name VARCHAR (50) NOT NULL,
+      email VARCHAR (100) NOT NULL,
+      birthdate date NOT NULL,
+      added timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+      PRIMARY KEY (id)
+    );
+    INSERT INTO authors VALUES ('1','Eileen','Lubowitz','ppaucek@example.org','1991-03-04','2004-05-30 02:08:25'),
```

Comment:
We had discussed the need to add a SQL to change owner here. Why is that no longer needed?

Reply:
The owner was already yugabyte. In the postgres-init file they were changing it expilicitly to lbuser. So I just removed the part where they changed the owner to lbuser. The default owner hence is yugabyte.